packages/core/src/layout/BottomLink/BottomLink.tsx

```diff
@@ -26,6 +26,7 @@ import {
 import ArrowIcon from '@material-ui/icons/ArrowForward';
 import { BackstageTheme } from '@backstage/theme';
 import Box from '@material-ui/core/Box';
+import { Link as InternalLink } from '../../components/Link';
```

Comment:
Afaik we should just always be using this one 😅 , it works for external links too right?

Reply:
Yes that was my first thought, too. But it didn't work as intended. Maybe I did something wrong 😅  I'm going to check that out.

Reply:
Ah yeah react-router doesn't handle that https://github.com/ReactTraining/react-router/issues/1147

I suggest we make the @backstage/core Link component detect whether it's an external link or not, and render accordingly

Reply:
Not sure... But since we are using the RouterLink of `react-router-dom`:
```
export const Link = React.forwardRef<any, Props>((props, ref) => (
  <MaterialLink ref={ref} component={RouterLink} {...props} />
));
``` 
the external links shouldn't work, right?

Reply:
@Rugvip I tried something out. But I am not sure if this is the best solution.